modules/virt-enlarging-vm-disk.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-vm-disk-enlargement.adoc
+
+[id="virt-enlarging-vm-disk_{context}"]
+= Enlarging a virtual machine disk
+
+A virtual machine (VM) disk has a default or initial storage capacity when you create it. However, you can enlarge the disk by resizing the disk's persistent volume claim (PVC).
+
+VM disk enlargement makes extra space available to the virtual machine. However, it is the responsibility of the VM owner to decide how to consume the storage.
+
+Enabling VM disk enlargement notifies the virtual machine about size changes and, if the disk is a Filesystem PVC, the matching file expands to the remaining size while reserving some space for file system overhead.
+
+.Procedure
+
+. Edit a VM disk's PVC using this command:
+
+[source, temrinal]
+----
+$ oc edit pvc my-disk-pvc
+----
+
+. Increase the `spec.resource.requests.storage` item to a larger size in the manifest of the VM disk you want to expand:
+
+[source,yaml]
+----
+spec:
```

Comment:
Is this a PVC manifest? It's a good idea to include the beginning of the manifest so that the `kind:` is visible.